knative-operator/hack/update-manifests.sh

```diff
@@ -37,8 +37,9 @@ function download_kafka {
 
 download_kafka knativekafka "$KNATIVE_EVENTING_KAFKA_VERSION" "${kafka_files[@]}"
 
-# Change the minavailable pdb for kafka-webhook to 1
-#git apply "$root/knative-operator/hack/007-eventing-kafka-pdb.patch"
+# Apply the patches
 
-# For 1.17 we still skip HPA/PDB
-git apply "$root/knative-operator/hack/001-remove_hpa_pdb.patch"
+for p in `ls $root/knative-operator/hack/*.patch|sort -n`
+do
+  git apply $p
```

Comment:
let's please not do this. thet;s keep them separate, and link to issues.

This was inherited from this hackery: 
https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/hack/update-manifests.sh#L78-L86

that way, we do have context!

Reply:
I did move the comment inside the diff file itself but sure, it's a separate issue

Reply:
yeah, same w/ renaming the patch files. LEt's do that if we want to afterwards